### PR TITLE
Module 2 project exercise

### PR DESCRIPTION
**Pull request by aniekanokpokpo**

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -2,17 +2,49 @@
...
+            <ul class="list-group">
+                {% for item in items %}
+                    {% if item.status == 'Not Started' %}
+                        <li class="list-group-item">{{ item.title }} <br> {{ item.desc }}
+                            <span class="badge badge-danger" style="float: right">
+                                <form method="post" action="{{ url_for('complete_item', id =item.id) }}">
+                                    <button class="btn btn-link" type="submit" data-toggle="tooltip"
+                                            title="Mark as completed">
+                                    </button>
+                                </form>{{ item.status }}
```
Comment (hugh-emerson) [inline templates/index.html]: I think you meant to put item.status inside the `<button>...</button>`
Currently the button doesn't have any text so it's not obvious it's there and clicking on the text does nothing.

File: app.py [hunk 2]
```diff
@@ -1,12 +1,31 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
```
Comment (hugh-emerson) [inline app.py:2]: You can delete the session_items.py file and this line

File: trello_items.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+    Args:
+        item: The item to save.
+    """
+    existing_items = get_items()
+    updated_items = [item if item['id'] == existing_item['id'] else existing_item for existing_item in existing_items]
+
+    # session['items'] = updated_items
+
+    return item
```
Comment (hugh-emerson) [inline trello_items.py]: This function is unused (and doesn't work).
It should be deleted.

File: trello_items.py [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    """
+    Fetches all saved items from the session.
+
+    Returns:
+        list: The list of saved items.
+    """
+    r = requests.get(f'https://api.trello.com/1/lists/{Config.LIST_ID}/cards?key={Config.KEY}&token={Config.TOKEN}')
+    done_request = requests.get(
+        f'https://api.trello.com/1/lists/{Config.DONE_LIST_ID}/cards?key={Config.KEY}&token={Config.TOKEN}')
+    response = r.json()
```
Comment (hugh-emerson) [inline trello_items.py]: Variable names are a bit off:

`r` is a response and `r.json()` is the response body.
`done_request` is actually a Response object.

More importantly, it's a bit confusing how you are treating the two requests slightly differently when they are doing the same thing for different lists. Would be neater to have something like:
```
not_started_response = requests.get(...)
done_response = requests.get(...)
```

**Review by hugh-emerson — APPROVED** [review]
The Trello integration looks good so you can merge when you're ready.

But you should not have deleted .env.template. In fact, you should keep it up to date with placeholder values for variables you've added. It is copied into .env when you first set up the project (see the README). You should also update README.md as you make changes to the project. It should contain everything the reader needs to know to get the project up and running after cloning it from GitHub.

See my other comments for some suggestions.

File: trello_items.py (hunk 3 above)
Comment (aniekanokpokpo) [reply trello_items.py]: deleted

**Pull request merged**